mobile/lib/main.dart

```diff
@@ -32,12 +33,15 @@ class MainApp extends StatelessWidget {
             bottomSize: 8,
```

Comment:
According to figma, this should be 5px high:

![obraz](https://github.com/RadioAktywne/ra-app/assets/35864414/2660b8f4-77f8-4dcf-b2f4-e602bc8160eb)

```suggestion
            bottomSize: 5,
```
It would be nice to change it now with the bottomnavbar adjustments.

Reply:
This one isn't actually mine - the RaAppBar was already on github (to be honest - the app bar having the border of size 8 is why mine has too).